### DISCUSSION TITLE
Import/Export

### DISCUSSION THREAD

**Discussion by kcoax** [outline/outline#2778] [General]
Hello,
 when Outline will support .doc or .html documents to import? 
Right now, the latest version can only support Outline's .md export right?

Kindest regards

**Comment by tommoor** [comment]
We already support both of these formats 👍

**Reply by kcoax** [reply]
Could you please show me how? As When I try to Import, it says Sorry, the file xxx.zip is missing valid collections or documents.For every single one, even for .md files

**Reply by tommoor** [reply]
Hi, it sounds like you're trying to go through the import/export in settings? This is for importing in bulk from another installation.

For regular imports you can drag into the sidebar or use the "Import document" option from document and collection menus
